models/Provider.php

```diff
@@ -89,6 +89,10 @@ protected static function registerUser(ProviderUser $providerUser, Provider $pro
             'email' => $providerUser->getEmail(),
             // Generate a random password for the new user
             'password' => str_random(16),
+            // Assign the new user to default group
+            'customer_group_id' => 1,
+            // New user enabled by default
+            'status' => 1,
```

Comment:
The `status` index here is redundant, you can pass true as the second argument to the `Auth::register($data, TRUE)` method to auto activate the customer and also set status to 1.

Nice first PR 👍